### PR TITLE
Added Altearn.xyz and BuildTheEarth.net

### PR DESCRIPTION
**Pull request by fantomitechno**
This time with Altearn.xyz with 2 mounth

> Proof it's working : 

Altearn.xyz 
without 
image
and with the privacy mode 
image
(This website was realy easy because with the website construction I can just see that page your looking with the window title)

BuildTheEarth.net
image
without 
image
and with Hide map position 
image
(I'm realy proud of the follow with the map)
image
and finaly with privacy mode 
image

### CODE REVIEW

**Review by Bas950 — CHANGES_REQUESTED** [review]
Why WebSite in the names of the apps, and altearn presence seems pretty low quality

**Comment by fantomitechno** [comment]
for the "Website", I modified this 

for the Altearn presence, I forget to update my github, I leave it in local ^^'
That was the first version and I forget my local version, I will update this tomorow.

**Comment by fantomitechno** [comment]
So that's now the true version : 
> Proof it work

image

image

image

image

image

image

image

File: websites/B/BuildTheEarth/presence.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    presenceData.details = "Viewing a page:";
+    presenceData.state = "Profil";
+  } else if (window.location.pathname.endsWith("faq")) {
+    presenceData.details = "Viewing a page:";
+    presenceData.state = "FAQ";
+  } else if (window.location.pathname.endsWith("map")) {
+    presenceData.smallImageKey = "map";
+    presenceData.details = "Viewing a page:";
+    presenceData.state = "Map : "+Math.floor(Number(document.getElementById('map_tpll_command').textContent.replace('/tpll ','').split(' ')[0])*10)/10+' | '+Math.floor(Number(document.getElementById('map_tpll_command').textContent.replace('/tpll ','').split(' ')[1])*10)/10;
+    if (sprivacy) presenceData.state = "Map";
```
Comment (ririxidev) [inline websites/B/BuildTheEarth/presence.ts:26]: fr?

File: websites/B/BuildTheEarth/presence.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  } else if (window.location.pathname.endsWith("contact")) {
+    presenceData.details = "Viewing a page:";
+    presenceData.state = "Contact";
+  } else if (window.location.pathname.endsWith("upload")) {
+    presenceData.details = "Viewing a page:";
+    presenceData.state = "Upload your world";
+  } else {
+    presenceData.details = "Viewing a page:";
+    presenceData.state = document.title.replace(' - BuildTheEarth','');
+    if (window.location.pathname.length != 1) presenceData.details = "Viewing a BuildTeams:";
```
Comment (ririxidev) [inline websites/B/BuildTheEarth/presence.ts:39]: fr?

**Review by ririxidev — CHANGES_REQUESTED** [review]

**Comment by fantomitechno** [comment]
Modified

File: websites/B/BuildTheEarth/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    "category": "games",
+    "settings": [
+      {
+        "id": "privacy",
+        "title": "Privacy mode",
+        "icon": "fad fa-user-secret",
+        "value": false
+      },
+      {
+        "id": "sprivacy",
```
Comment (Bas950) [inline websites/B/BuildTheEarth/dist/metadata.json:29]: please give the ID a better name
something like mappos or something along those lines

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Pull request merged**